run.py

```diff
@@ -13,29 +14,16 @@
 the given module or script.
 """
 
-import os
-import sys
 import imp
```

Comment:
Nope.. They were forgotten there. Just run `run.py` without them and there were no issues... Shall I push the changes?